larq/metrics.py

```diff
@@ -87,18 +135,24 @@ def __init__(
         super().__init__(name=name, dtype=dtype)
         self.values_dtype = tf.as_dtype(values_dtype)
         self.values_shape = tf.TensorShape(values_shape).as_list()
+        self.is_weight_metric = True
         with tf.init_scope():
             self._previous_values = self.add_weight(
                 "previous_values",
                 shape=values_shape,
                 dtype=self.values_dtype,
                 initializer=tf.keras.initializers.zeros,
+                aggregation=tf.VariableAggregation.SUM,
```

Comment:
I am not sure if this is correct. Could you elaborate on this change?

Reply:
You already had it on SUM before, it was the default value in the add_weight function. However, SUM may not always be the desired aggregation method, so I changed the default to None (which will throw an error if you run it on multi-gpu, rather than computing incorrect values) and explicitly set it back to SUM here. 
That being said, I think it is actually incorrect here; I don't think FlipRatio can support multi-GPU at all. In that case, I should set it to None. What do you think?

Reply:
Alternatively, we could set all of these to ONLY_FIRST_REPLICA, which will use only one of the results in the multi-gpu case and ignore the others.

Reply:
I changed it to ONLY_FIRST_REPLICA in the latest commit. Since the FlipRatio uses weights instead of activations, I assume that even across multiple GPU's, either the metric would only be computed once, or all replicas would compute the same value.

Reply:
I think switching to `ONLY_FIRST_REPLICA` makes sense here :+1: